frontend/src/_helpers/handle-response.js

```diff
@@ -28,3 +28,13 @@ export function handleResponseWithoutValidation(response) {
     return data;
   });
 }
+
+export const handleErrConnections = (error) => {
```

Comment:
shall we do this for all APIs using some global configs, we can do it in different PR

Reply:
Sure. I will remove this Part